lib/bundles/inspec-compliance/api.rb

```diff
@@ -276,15 +280,21 @@ def self.target_is_automate_server?(url, insecure)
             "Received 200 from #{url}#{automate_endpoint} " \
             'assuming target is an OpsWorks Chef Automate instance',
           )
-          return true
+          true
         else
           Inspec::Log.debug(
             "Received 200 from #{url}#{automate_endpoint} " \
             'but did not receive the Chef Manage page. ' \
             'Continuing with detection attempts',
           )
-          return false
+          false
         end
+      else
+        Inspec::Log.debug(
+          "Received neither 200 nor 401 from #{url}#{automate_endpoint}. " \
+          'Continuing with detection attempts',
```

Comment:
This assumes that whatever called this method has additional detection attempts to make :)

A better message, to keep it contained, would be `"Received unexpected status code #{response.code} from #{url}#{automate_endpoint} - assuming this is not an Automate server."`

... and whoever is calling can determine whether there are additional detection attempts to make, and now the user gets to see what status code they actually got.

Reply:
Ah, great catch. Fixing now.